src/tools/generate_workspace/src/main/java/com/google/devtools/build/workspace/GenerateWorkspace.java

```diff
@@ -73,6 +80,7 @@ public static void main(String[] args) {
   }
 
   private static void printUsage(OptionsParser parser) {
+    // TODO(mickey): Update doc for new output flags.
```

Comment:
Might as well add to https://github.com/bazelbuild/bazel/blob/master/site/docs/external.md now.